pkgs/by-name/ho/hoppscotch/package.nix

```diff
@@ -1,30 +1,35 @@
-{ lib
-, stdenv
-, fetchurl
-, appimageTools
-, undmg
+{
+  lib,
+  stdenv,
+  fetchurl,
+  appimageTools,
+  undmg,
+  copyDesktopItems,
```

Comment:
Unused import.
```suggestion
```

Reply:
https://github.com/NixOS/nixpkgs/pull/343315, pre-existing issue, but i fixed it too